src/components/NewOrgDelegationPage/NewOrgDelegationPage.module.css

```diff
@@ -0,0 +1,59 @@
+@media only screen and (min-width: 0px) {
+  .page{
+      --pageContainer-width: 1055px;
+  }
+}
+
+@media only screen and (min-width: 700px) {
+  .page{
+    --pageContainer-width: 1055px;
```

Comment:
Why specify for smaller screens if the widths are all the same anyway?

Reply:
Because we have to write these exact media queries in a few weeks :P

Reply:
Ahaaaa... So they're just placeholders then?

Reply:
Yup, you can choose if we should keep or remove them.

Reply:
They might as well be there 🙂